content/swiftui/concepts/viewmodifier/terms/foregroundColor/foregroundColor.md.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+Title: .foregroundColor()
+Description: The .foregroundColor() modifier sets the foreground color displayed in a view.
+Subjects:
+ - 'SwiftUI'
+Tags: 
+ - 'SwiftUI'
+ - 'Modifier'
+ - 'View'
+ CatalogContent:
+ - 'learn-swiftUI'
+ - docs/content/swiftui/concepts/viewmodifier/terms/foregroundColor
+---
+# .foregroundColor ()
+The `.foregroundColor()` modifier sets the foreground color displayed in a [View](https://www.codecademy.com/resources/docs/swiftui/views).
+
+
+
+## Syntax
+
+
+```
+struct MyView: View {
+		 var body: some View {
+			 Text("We are setting the foreground color to green!")
+			 .foregroundColor(.green)
+	 }
+ }
+ ```
+
+The `.foregroundColor(green)`  is applying the color green to the `Text` (which conforms to the `View` protocol). 
```

Comment:
```suggestion
The `.foregroundColor(green)` is applying the color green to the `Text` (which conforms to the `View` protocol). 
```

Reply:
I have made these changes and committed the updated file. This has been pushed. Thank you!